src/main/msp/msp.c

```diff
@@ -2149,6 +2157,15 @@ static mspResult_e mspProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
             sbufReadU8(src);
             sbufReadU8(src);
             sbufReadU8(src);
+#endif
+        }
+        if (sbufBytesRemaining(src) >= 2) {
```

Comment:
@mikeller looking at this, maybe is better to include this inside the upper if? Both has been added in this version.

Reply:
Yes please, well spotted. And we should also continue with adding annotations about the version something has been added in, like https://github.com/betaflight/betaflight/pull/7725/files#diff-eefd576c86ccfa8b6cdf98081806a7a8L2103

Reply:
Done.

Reply:
Thanks! I can approve it thought...